auth/auth.go

```diff
@@ -119,7 +118,11 @@ func (n *NonceHandler) Nonce(
 		n.nonceCache = make(map[string]uint64)
 	}
 	var seq uint64
-	if state.FeatureEnabled(features.IncrementNonceOnFailedTxFeature, false) && !isCheckTx {
+
+	cfg := state.Config()
+	IncrementNonceOnFailedTx := cfg.GetNonceHandler().GetIncNonceOnFailedTx() > 0
```

Comment:
`IncrementNonceOnFailedTx` -> `incrementNonceOnFailedTx`
And why do we need `cfg` if you only access it once?

Reply:
Fixed